resources/main.mjs

```diff
@@ -290,6 +291,36 @@ class MainBenchmarkClient {
         csvLink.setAttribute("download", `${filePrefix}.csv`);
     }
 
+    _populateNonStandardParams() {
+        if (params === defaultParams)
+            return;
+        const paramsDiff = [];
+        const usedSearchparams = params.toSearchParamsObject();
+        const defaultSearchParams = defaultParams.toCompleteSearchParamsObject(false);
+        for (const [key, value] of usedSearchparams.entries()) {
+            if (key === "developerMode")
+                continue;
+            const defaultValue = defaultSearchParams.get(key);
+            if (value !== defaultValue)
+                paramsDiff.push({ key, value, defaultValue });
+        }
+        if (paramsDiff.length === 0)
+            return;
+        let body = "";
+        for (const { key, value, defaultValue } of paramsDiff)
+            body += `<tr><th>${key}</th><th>${value}</th><th>${defaultValue}</th></tr>`;
+        const table = document.getElementById("non-standard-params-table");
+        table.innerHTML = `<thead>
```

Comment:
It's probably better to have this in the HTML side and only replaced the contents of tbody.

Reply:
ah yes, much nicer that way. 👍